modules/compliance-operator-configure-scanning.adoc

```diff
@@ -13,7 +13,7 @@ Create a `ScanSettingBinding` object in the `openshift-compliance` namespace to
 ====
 * If you are using the compliance 2.0 feature, you can schedule your scan by using {product-title-short} to create a compliance scan schedule instead of creating the `ScanSettingBinding` on the Compliance Operator.
 +
-For more information about scheduling a compliance scan by using the compliance 2.0 feature, see "Creating a compliance scan schedule" in the "Additional resources” section.
+For more information about scheduling a compliance scan by using the compliance 2.0 feature, see "Customizing and automating your compliance scans" in the "Additional resources” section.
```

Comment:
For future PRs, the guidance now is to not use the "in the Additional resources section" part. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations

Reply:
Thanks for calling this out. I will create a separate task to address this across ACS documentation.